File: concrete/src/main/java/pcb/uwu/unit/composite/electromagnetism/SiemensUnit.java
package pcb.uwu.unit.composite.electromagnetism;

import pcb.uwu.units.composite.electromagnetism.ElectricConductanceUnit;

import static pcb.uwu.unit.quantity.AmpereUnit.AMPERE;
import static pcb.uwu.unit.quantity.KiloGramUnit.KILOGRAM;
import static pcb.uwu.unit.quantity.MeterUnit.METER;
import static pcb.uwu.unit.quantity.SecondUnit.SECOND;

public class SiemensUnit extends ElectricConductanceUnit {

	public static final SiemensUnit SIEMENS = new SiemensUnit();

	public SiemensUnit() {
		super(KILOGRAM, METER, AMPERE, SECOND);
	}

	@Override
	public String getSymbol() {
		return "S";
	}

	@Override
	public String getSingularName() {
		return "siemens";
	}

	@Override
	public String getPluralName() {
		return "siemens";
	}
}

File: concrete/src/main/java/pcb/uwu/unit/derived/electromagnetism/SiemensUnit.kt
package pcb.uwu.unit.derived.electromagnetism;

import static pcb.uwu.unit.base.AmpereUnit.AMPERE;
import static pcb.uwu.unit.base.KiloGramUnit.KILOGRAM;
import static pcb.uwu.unit.base.MeterUnit.METER;
import static pcb.uwu.unit.base.SecondUnit.SECOND;

public class SiemensUnit extends ElectricConductanceUnit {

	public static final SiemensUnit SIEMENS = new SiemensUnit();

	public SiemensUnit() {
		super(KILOGRAM, METER, AMPERE, SECOND);
	}

	@Override
	public String getSymbol() {
		return "S";
	}

	@Override
	public String getSingularName() {
		return "siemens";
	}

	@Override
	public String getPluralName() {
		return "siemens";
	}
}
